embassy-net-adin1110/.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
Individual crates in the repo shouldn't have their own "scaffolding", can you remove this?

Reply:
Yes, I shall remove them.